### DISCUSSION TITLE
Autocompletion on text search

### DISCUSSION THREAD

**Discussion by joshuarichards001** [supabase/supabase#13160] [Questions]
Can autocompletion be done when searching a column name using the textSearch?

If not are there any plans for this to be implemented?

Cheers,
Josh

**Comment by GaryAustin1** [comment]
I do auto complete with ilike and wildcard %.  This works well for titles.  It may not scale to millions of rows though.  I set a limit to number of matches with an indicator there are more and start after the 3rd character. 

Full text search is not really designed for searching for one or two characters, it is more about pattern matching.

"Any plans", not sure what you mean by plans.  textSearch is just using Postgres full text search capability.
